locale/en/docs/guides/debugging_getting_started.md

```diff
@@ -0,0 +1,198 @@
+---
+title: Debugging - Getting Started
+layout: docs.hbs
+---
+
+# Debugging Guide
+
+The goal of this guide is to provide you with enough information to get started
+debugging your Node.js apps and scripts.
+
+## Enable Debugging
+
+### --debug
+
+When started with the **--debug** or **--debug-brk** switches, Node.js listens
+for debugging commands defined by the [V8 Debugging Protocol][] on a TCP port,
+by default `5858`. Any debugger client which speaks this protocol can connect
+to and debug the running process.
+
+A running Node.js process can also be instructed to start listening for
+debugging messages by signaling it with `SIGUSR1` (on Linux and OS X).
```

Comment:
Thanks for the suggestions, I'm going to gather the --debug info in its own section toward the end.

@naugtur: the markdown headers get converted into HTML anchors ([fragment identifiers](https://www.w3.org/DesignIssues/Fragment.html), so it should be possible to refer directly to the `--debug` section. I'm not going to make the content dynamic based on the fragment here (too much work at the moment), but if you think that would be useful you could start another PR. Thanks!